### ISSUE TITLE
Migration Customization

### ISSUE THREAD

**Issue by antonkomarev** [cybercog/laravel-love#71]
There were issues with published migrations raised in #28. We need to think about the way to ignore default migrations to be executed.

### 1. Use global Love facade

If you are not going to use Love's default migrations, you should call the `Love::ignoreMigrations` method in the register method of your `AppServiceProvider`. You may export the default migrations using the `php artisan vendor:publish --tag=love-migrations` command and make changes before `php artisan migrate` execution.

### 2. Use configuration file

If you are not going to use Love's default migrations, you should export default config file using the `php artisan vendor:publish --tag=love-config` and change `love.load_default_migrations` boolean flag value to `false`. You may export the default migrations using the `php artisan vendor:publish --tag=love-migrations` command and make changes before `php artisan migrate` execution.

**Comment by antonkomarev** [comment]
Is there any idea why Taylor decided to implement is using facade instead of boolean flag in configuration file?

**Comment by JFunu** [comment]
@antonkomarev , I am not sure why he used that approach but I think using a boolean flag in the configuration file might be the best approach, since it becomes immediately obvious to the user, the options that can be turned on or off.

**Comment by antonkomarev** [comment]
This feature published in [v7.2](https://github.com/cybercog/laravel-love/releases/tag/7.2.0).

If you are not going to use Love's default migrations, you should export default config file.

```
$ php artisan vendor:publish --tag=love-config
```

Change `load_default_migrations` boolean flag value to `false` in published `config/love.php` file.

Export the default migrations using the `vendor:publish` Artisan command.

```
$ php artisan vendor:publish --tag=love-migrations
```

Make changes in published files and migrate them.

```
$ php artisan migrate
```
